linux/ubuntu/20.04/foxpass_setup.py

```diff
@@ -213,6 +219,52 @@ def fix_nsswitch():
     os.system("sed -i 's/shadow:.*/shadow:         compat ldap/' /etc/nsswitch.conf")
 
 
+def write_ldap_sudoers(uris, basedn, binddn, bindpw, sudoers_timed, bind_timelimit, query_timelimit):
+    check_sudo_passwd()
+    return_code = os.system('DEBIAN_FRONTEND=noninteractive apt-get install -y sudo-ldap')
+    if return_code != 0:
+        # bitshift right 8 to get rid of the signal portion of the return code
+        sys.exit(return_code >> 8)
+    with open('/etc/sudo-ldap.conf', "w") as w:
+        content = """\
+#
```

Comment:
Suggestion: I think it would be useful to show the user the changes that were made to the configuration files. It could be an optional one and can be based on user inputting `yes` for a message `would you like to know the details of the modifications made by the script`